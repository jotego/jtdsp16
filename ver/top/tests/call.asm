r0=1
r2=2
r3=3
call change

loop:
rb=0xcafe
goto loop

change:
a0=pr
a1=pi
r0=4
r2=5
r3=6
return
