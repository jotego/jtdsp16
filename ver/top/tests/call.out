r0=0x0004
r1=0x0000
r2=0x0005
r3=0x0006
rb=0xcafe
re=0x0000
j =0x0000
k =0x0000
pr=0x0004
pt=0x0000
i =0x0000
a0=0x000000000
a1=0x000000000
